CHANGELOG.md

```diff
@@ -21,6 +21,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   [cyberark/conjur-api-python3#89](https://github.com/cyberark/conjur-api-python3/issues/89)
 
 ### Changed
+- The .conjurrc parameters have been renamed from `account` to `conjur_account` and `appliance_url` to `conjur_url`.
```

Comment:
@orenbm 👀 ?

Reply:
... and from `appliance_url` to `conjur_url`